sources/web/datalab/polymer/modules/bigquery-file-manager/bigquery-file-manager.ts

```diff
@@ -41,6 +41,10 @@ class BigQueryFile extends DatalabFile {
  */
 class BigQueryFileManager extends BaseFileManager {
 
+  protected myFileManagerType() {
```

Comment:
nit: I thought protected methods should appear after public ones?

Reply:
Done.